server/src/main/java/org/apache/iotdb/db/engine/memtable/WritableMemChunk.java

```diff
@@ -223,51 +223,6 @@ public void setTimeOffset(long offset) {
     list.setTimeOffset(offset);
   }
 
-  @Override
-  public synchronized List<TimeValuePair> getSortedTimeValuePairList() {
```

Comment:
remove the sortedList field in this class and unused import

Reply:
done